src/app/globals.css

```
@@ -69,24 +81,39 @@
+/* Force solid background on sidebar trigger */
+[data-sidebar-trigger] {
+  background-color: var(--color-primary) !important;
+  opacity: 1 !important;
+  -webkit-backdrop-filter: none !important;
+  backdrop-filter: none !important;
+}
 
+/* Specifically target the sidebar trigger button */
+button[data-sidebar-trigger] {
+  background-image: none !important;
+  background: var(--color-primary) !important;
```

Biome (2.1.2)

[error] Unexpected shorthand property background after background-image



(lint/suspicious/noShorthandPropertyOverrides)